src/SkyDrop.iOS/Common/iOSUtil.cs

```diff
@@ -17,5 +22,44 @@ public static Task<UIImage> BitMatrixToImage(BitMatrix bitMatrix)
                 return renderer.Render(bitMatrix, ZXing.BarcodeFormat.QR_CODE, "");
             });
         }
+
+        public static void RegisterForLocalNotifications()
+        {
+            var settings = UIUserNotificationSettings.GetSettingsForTypes(UIUserNotificationType.Alert, null);
+            UIApplication.SharedApplication.RegisterUserNotificationSettings(settings);
+        }
+
+        public static void ShowUploadFinishedNotification(FileUploadResult uploadResult, string filename)
+        {
+            switch (uploadResult)
+            {
+                case FileUploadResult.Success:
+                    ShowNotification("File published successfully (tap to view)", filename);
```

Comment:
Does tapping the notif show the QR code?

Reply:
If the app is running in the background it will show the QR code, if the app is not running it will show send receive buttons

Reply:
Should it show the QR code regardless? Seems like the correct behaviour imo